### PR TITLE
RAD-31 Added support for super-bias reference files.

### PR DESCRIPTION
**Pull request by PaulHuwe**
Added support for super-bias reference files.

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
Do we need the MA table info included?

File: src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+type: object
+properties:
+  meta:
+    allOf:
+      - $ref: ref_common-1.0.0
+      - type: object
+        properties:
+          reftype:
+            type: string
+            enum: [DARK]
```
Comment (gardner358) [inline src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml:17]: Shouldn't this be superbias?

File: src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
+    datatype: float32
+    ndim: 2
+  dq:
+    title: 2-D data quality array for all planes
+    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
+    datatype: uint32
+    ndim: 2
+  err:
+    title: Error array
```
Comment (gardner358) [inline src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml:29]: Do you want 2-D before the Error array?

File: src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml:17]: Oops, should be bias. Fixing.

File: src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml (hunk 2 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/reference_files/superbias-1.0.0.yaml:29]: Sure.

**Review by gardner358 — APPROVED** [review]

**Comment by PaulHuwe** [comment]
We are discussing the MA Table / mode implementation offline.

**Comment by PaulHuwe** [comment]
INS / DMS group is still deciding on the remaining Bias particulars, so I will commit this for now.

**Pull request merged**